src/TheCallCenter/Startup.cs

```diff
@@ -24,8 +21,7 @@ public Startup(IConfiguration config)
     // This method gets called by the runtime. Use this method to add services to the container.
     public void ConfigureServices(IServiceCollection services)
     {
-      services.AddDbContext<CallCenterContext>()
-        .AddEntityFrameworkSqlServer();
+      services.AddDbContext<CallCenterContext>(options => options.UseInMemoryDatabase("WilderMinds_SignalRByExample"));
```

Comment:
Unless you did something on purpose here, the database configured would work on any Visual Studio version (a dev-time DB is installed by default).  While moving this to in-memory makes sense, there are side effects that may cause others problems.

Reply:
I managed to get it working -- it is not inherently there by default and you need a minor amount of knowledge with EF to get it working. I've added some step-by-step instructions to the ReadMe in the next upcoming change to this pull request